services/course-material/src/components/ContentRenderer/core/common/Paragraph/index.tsx

```
@@ -1,172 +1,59 @@
-
-        ${dropCap ? hasDropCap : null}
-      `}
+      className={getParagraphStyles(
+        textColor,
+        backgroundColor,
+        fontSize,
+        hideOverflow,
+        dropCap,
+        align,
+      )}
       dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)